Source/WarpX.H

```diff
@@ -1197,6 +1199,7 @@ private:
 
     int max_step   = std::numeric_limits<int>::max();
     amrex::Real stop_time = std::numeric_limits<amrex::Real>::max();
+    int terminate_flag = 0;
```

Comment:
```suggestion
    volatile int terminate_flag = 0;
```
So that the compiler doesn't optimize out the load

Reply:
please add a doxygen string to the member variable, too :)
```suggestion

    /** System signal handling: when set, terminate the simulation in the next step */
    volatile int terminate_flag = 0;
```

Reply:
Oh, I realize this should be `sig_atomic_t` from `#include <csignal>`